spec/features/course/forum/post_management_spec.rb

```diff
@@ -162,25 +159,19 @@
         visit course_forum_topic_path(course, forum, topic)
         # Mark as answer
         within find("div.post_#{post.id}") do
-          find('svg[data-testId="CheckCircleOutlineIcon"]').find(:xpath, '..').click
+          find('span', text: 'Mark as answer').find(:xpath, '../..').click
+          expect(page).to have_selector('span', text: 'Unmark as answer')
```

Comment:
Consider using `have_text('Unmark as answer')`? Then if the HTML element changes, this test will stay resilient.